server/src/test/java/org/elasticsearch/search/SearchCancellationTests.java

```diff
@@ -53,17 +57,19 @@ public static void setup() throws IOException {
         // we need at least 2 segments - so no merges should be allowed
         w.w.getConfig().setMergePolicy(NoMergePolicy.INSTANCE);
         w.setDoRandomForceMerge(false);
-        indexRandomDocuments(w, TestUtil.nextInt(random(), 2, 20));
+        int numDocs = nextInt(random(), 2, 20);
+        indexRandomDocuments(w, numDocs, 0);
         w.flush();
-        indexRandomDocuments(w, TestUtil.nextInt(random(), 1, 20));
+        indexRandomDocuments(w, nextInt(random(), 1, 20), numDocs);
```

Comment:
I tend to prefer fully-qualified calls to `TestUtil#nextInt` instead of adding a static import, given that this method name is quite generic.